torchgen/api/python.py

```diff
@@ -998,17 +998,36 @@ def returns_named_tuple_pyi(signature: PythonSignature) -> Optional[Tuple[str, s
     namedtuple_name = signature.name
     field_names = namedtuple_fieldnames(signature.returns.returns)
     if field_names:
-        namedtuple_def_lines = [f"class {namedtuple_name}(NamedTuple):"]
+        # These types are structseq objects which act like named NamedTuples, but
```

Comment:
nit: could we change the function name to `returns_structseq_pyi` or something similar?